integration_test/mysql/all_test.exs

```diff
@@ -2,7 +2,7 @@ Code.require_file "../sql/escape.exs", __DIR__
 Code.require_file "../sql/lock.exs", __DIR__
 Code.require_file "../sql/migration.exs", __DIR__
 Code.require_file "../sql/sql.exs", __DIR__
-Code.require_file "../sql/test_transaction.exs", __DIR__
+# Code.require_file "../sql/test_transaction.exs", __DIR__
```

Comment:
I would say we still need to have those tests, we just need to rewrite them with ownership.